recipes/sdl2/run_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+${PREFIX}/bin/sdl2-config --version
```

Comment:
True, but afaik sdl-config only works on unix systems. The Windows builds don't come with something like that (at least, the executable is not built and installed when building with CMake)